CityOfDetroit/bloom
# fix: adding home type to model

# Pull Request Template

## Issue Overview

This PR addresses #1479 


## Description

This PR adds home types to the listing model and handles updating/viewing this information on the partner's side.

## How Can This Be Tested/Reviewed?

This can be reviewed by logging into the partner's side and then selecting a listing. View the home type in the Unit Section, click edit, update it, and ensure that the new information is reflected in that same Unit Section.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
